core/src/main/java/com/zextras/carbonio/files/dal/repositories/impl/ebean/FileVersionRepositoryEbean.java

```diff
@@ -98,16 +98,27 @@ public Optional<FileVersion> createNewFileVersion(
   }
 
   @Override
-  public List<FileVersion> getFileVersions(String nodeId) {
-
-    List<FileVersion> fileVersions = mDB.getEbeanDatabase()
-      .find(FileVersion.class)
-      .where()
-      .eq(Files.Db.FileVersion.NODE_ID, nodeId)
-      .order()
-      .desc(Db.FileVersion.VERSION)
-      .findList();
-
+  public List<FileVersion> getFileVersions(String nodeId, boolean asc) {
```

Comment:
Right now this method is always called by production code with the asc to `false. I would not add this flag only for testing purpose (see comment on DatabasePopulator). I will specify the default order on the method name.

In the future, we will make a refactor when we will need another type of sorting